cutevariant/gui/plugin.py

```diff
@@ -28,6 +28,7 @@ class QueryPluginWidget(PluginWidget):
 
     #  signals
     changed = Signal()
+    message = Signal(str)
```

Comment:
what is the purpose ?

Reply:
When the plugin emits a message to be displayed in the status bar